cfg/pgrapher/common/sim/nodes.jsonnet

```diff
@@ -64,7 +64,7 @@ function(params, tools)
         },
     }, nin=1,nout=1,uses=[tools.random, anode] + pir_trio),
     
-    make_depotransform :: function(name, anode, pirs) g.pnode({
+    make_depotransform :: function(name, anode, plane, pirs) g.pnode({
```

Comment:
How about I make it a separate function? It'd be a copy that includes "plane" with a different name. My hope this that this would be usable for SBND (and DUNE) places where there are wrapped wires will have to deal with effects like this

Reply:
It would be legitimate to extend the function signature like:

```
function(name, anode, pirs, planes=[0,1,2])
```

In this way, old code which does not include `planes` will get `[0,1,2]` which is coincident with the default set in the C++.